objects/LuaScript.ttslua

```diff
@@ -1730,75 +1735,74 @@ function setupCampaignMisc()
 
   -- place the chapter 6 encounter deck on the board
   campaignBag.takeObject({
-    position = { 18.5, 1.54, 47 },
-    guid     = '0ac1dc',
-    smooth   = false,
-    rotation = { 0, 180, 180 }
+    position          = {18.5, 1.54, 47.00},
+    guid = '0ac1dc',
+    smooth = false,
+    rotation = {0.00, 180.00, 180.00}
   })
 
-  -- place the campaign book at the edge of the table
+  --place the campaign book at the edge of the table
   campaignBag.takeObject({
-    position          = { -27.86, 1.50, 50.34 },
-    smooth            = true,
-    guid              = '0cf9c7',
+    position          = {-27.86, 1.50, 50.34},
     callback_function = function(x)
       x.setLock(true)
-      x.setPositionSmooth({ 36, 6.8, 55 })
-      x.setRotationSmooth({ 60, 180, 0 })
-    end
+      x.setPositionSmooth({36.00, 6.80, 55.00})
+      x.setRotationSmooth({60.00, 180.00, 0.00})
+    end,
+    smooth            = true,
+    guid              = '0cf9c7',
   })
 
   --place the campaign tracker at the edge of the table
   campaignBag.takeObject({
-    position          = { -27.86, 1.5, 50.34 },
-    smooth            = true,
-    guid              = 'c64617',
+    position          = {-27.86, 1.50, 50.34},
     callback_function = function(x)
       x.setLock(true)
-      x.setPositionSmooth({ 45, 6.8, 55 })
-      x.setRotationSmooth({ 60, 180, 0 })
-    end
+      x.setPositionSmooth({45.00, 6.80, 55.00})
+      x.setRotationSmooth({60.00, 180.00, 0.00})
+    end,
+    smooth            = true,
+    guid              = 'c64617',
   })
 
-  -- place the overview cards in two rows below the campaign book
-  local overViewGuids = { '7e55f6', '2f7531', '248eeb', 'f1bd54', '78cbcd', '74971a' }
-  local startLoc = { 33, 1.55, 47 }
+  --place the overview cards in two rows below the campaign book
+  local overViewGuids = {'7e55f6', '2f7531','248eeb','f1bd54','78cbcd','74971a'}
+  local startLoc = {33.0, 1.55, 47.0}
   local newLoc = {}
-  for i, g in ipairs(overViewGuids) do
-    if i <= 3 then
-      newLoc = { startLoc[1] + (8 * (i - 1)), startLoc[2], startLoc[3] }
+  for i,g in ipairs(overViewGuids) do
+    if i <=3 then
+      newLoc = {startLoc[1]+(8*(i-1)), startLoc[2], startLoc[3]}
     else
-      newLoc = { startLoc[1] + (8 * (i - 4)), startLoc[2], 40 }
+      newLoc = {startLoc[1]+(8*(i-4)), startLoc[2], 40}
     end
     campaignBag.takeObject({
-      position = newLoc,
-      rotation = { 0, 180, 180 },
-      smooth   = false,
-      guid     = g
+      position          = newLoc,
+      rotation          = {0.00, 180.00, 180.00},
+      smooth            = false,
+      guid              = g,
     })
   end
 
   -- If two players are playing place the pidgeon tokens on the board
   if #seatedPlayerColors == 3 then
     campaignBag.takeObject({
-      position = { -15, 1.51, 36.50 },
-      smooth   = true,
-      rotation = { 0, 180, 180 },
-      guid     = '20c269'
+      position          = {-15.00, 1.51, 36.50},
```

Comment:
I think I remember changing this? Make sure to refresh the `main` branch before creating a new one. For this existing branch, please merge `main` into it to make sure it's up to date

Reply:
Hmm I did sync with the main branch before I pulled it down to the local repo. Let me check somethings.

Reply:
From what I'm seeing my bugfix branch is not behind the upstream branch, that should mean its synced right?

Reply:
That's correct. In that case I don't understand these changes though

Reply:
I guess that's part of some auto-formatting?

Reply:
Yeah I think it must be the editor spacing it out, these parts of the code wasn't touched.